md/customize-living-application-layout.md

```diff
@@ -86,55 +86,87 @@ Into the UI Designer:
+7. Resize the _living Application Menu_ by setting the width to _2_.
+8. Resize the _living Application IFrame_ by setting the width to _10_.
 9. Click on **Save**.
 
 ## Export the Side menu layout on your file system
 
 Once your changes are made, save the new layout using a new name and then export it.
 
-1. Rename it into **SideMenuLayout**.
+1. Rename it into _SideMenuLayout_.
 2. Click on **Save**.
 3. Click on the **Export** button.
 
-## Import the **Side Menu Layout** into the portal
+## Import the _Side Menu Layout_ into the portal
 
-1. Open the Bonita Portal, and again log in with username and password of someone mapped to the Administrator profile.
+1. Open Bonita Portal, and again log in with username and password of someone mapped to the Administrator profile.
 2. Click on the **Resources** menu.
 3. Click on **Add**.
-4. Add the new **Side Menu Layout Page**.
+4. Add the new _Side Menu Layout Page_.
 5. Click **Next**.
 6. Click **Confirm**.
 
 ## Configure your living application to use the modified layout
 
 1. Click on the **Applications** menu.
 2. Click on the edit action **...** of your living application.
-3. In the **Look & Feel** section, click on the layout link
-4. Select the **Side Menu layout**
+3. In the **Look & Feel** section, click on the **Layout** link
+4. Select the _Side Menu layout_
 5. Click on the validation button
-6. In the application details section, click on the URL link to navigate to your living application.
+6. In the **Application details** section, click on the URL link to navigate to your living application.
 7. See your new application layout with a side menu.
 8. Feel free to add lots of new improvements to create the layout that fits your needs.
 
+## Take advantage of the latest improvement in a customized layout
+
+<a id="improve-navigation"/>
+
+### Navigate between pages without reloading the entire page.
+
+Starting with Bonita 7.6.0, when the user clicks on any menu item, only the content of the iframe with the targeted page is refreshed: it does not reload the entire page anymore.  
+To take advantage of this improvement if your customized layout has been created with a Bonita version older than 7.6.0, you will need to follow these steps:
+
+1. Import your _CustomLayout_7.5.x_ (or lower) in the UI Designer 7.6.0 (or later versions)
+2. Export the _Default layout_ from Bonita Portal 7.6.0 (or later versions)
+3. Import the _Default layout_ in the UI Designer
+4. Open your _CustomLayout_7.5.x_ 
+5. If any of the custom widgets _livingApplicationMenu_ or _livingApplicationIFrame_ has been modified,  
+you will need to merge your modifications and the modifications of the new version of those widgets, namely _livingApplicationMenuV3_ and _livingApplicationIFrameV3_.  
+
+Note 1: To help this merge, you can generate a diff, between two version of the default layout by using Git.
+You will need to install Git, then clone the repository "https://github.com/bonitasoft/bonita-distrib.git".
```

Comment:
If you don't already have Git(link to git-scm.com), you will need to install it, then clone...